account_bank_statement_import_multi/wizards/account_bank_statement_import_multi.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import base64
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError, UserError
+
+
+class AccountBankStatementImportMulti(models.TransientModel):
+
+    _name = 'account.bank.statement.import.multi'
+
+    data_file = fields.Binary(
+        string='Bank Statements File',
+        required=True,
+    )
+    filename = fields.Char()
+
+    @api.multi
```

Comment:
don't need `api.multi` (removed in V13)

Reply:
but PR is for v11?

Reply:
yes, but in V10,V11,V12 `@api.multi` is not required (like that we will keep time during V13 migration)

Reply:
Right, yet for v11 it's not an issue and during v13 migration it will be removed. So it's up to author if to keep it or remove it, technically.